packages/web-components/src/components/table-of-contents/table-of-contents.ts

```diff
@@ -275,9 +282,20 @@ class DDSTableOfContents extends StableSelectorMixin(LitElement) {
       _handleSlotChange: handleSlotChange,
       _handleSlotChangeHeading: handleSlotChangeHeading,
     } = this;
+
+    const containerClasses = classMap({
+      [`${ddsPrefix}-ce--table-of-contents__container`]: this.type === TOC_TYPES.DEFAULT,
+      [`${ddsPrefix}-ce--table-of-contents-horizontal__container`]: this.type === TOC_TYPES.HORIZONTAL,
+    });
+
+    const navigationClasses = classMap({
+      [`${prefix}--tableofcontents__sidebar`]: this.type === TOC_TYPES.DEFAULT,
+      [`${prefix}--tableofcontents__horizontal`]: this.type === TOC_TYPES.HORIZONTAL,
```

Comment:
`__navbar` or `__topbar`? Naïvely I thought of suggesting `--horizontal` as "horizontal" means a style variant.